run-review-bot.py

```diff
@@ -1,17 +1,19 @@
 from lib import test_pull_request
 import argparse
+import sys
 
 
 def main():
     parser = argparse.ArgumentParser()
-    parser.add_argument('--pr-url', required=True)
     parser.add_argument('--base-sha', required=True)
     parser.add_argument('--current-sha', required=True)
-    parser.add_argument('--token', required=True, help='The GITHUB token for fetching package data')
     args = parser.parse_args()
-    res = test_pull_request(args.pr_url, args.base_sha, args.current_sha, args.token)
-    if res['result'] == 'error':
-        raise Exception(res['message'])
+    res = test_pull_request(args.base_sha, args.current_sha)
```

Comment:
For readability would do:

```suggestion
    exit_code, message = test_pull_request(args.base_sha, args.current_sha)
```